BridgeAppSDK/SBAGenericStepViewController.swift

```diff
@@ -79,13 +79,16 @@ open class SBAGenericStepViewController: ORKStepViewController, UITableViewDataS
     open var headerView: SBAStepHeaderView?
     open var navigationView: SBAStepNavigationView?
     
-    // we keep a copy of the original result after initialization so we can return that if the user
+    // We keep a copy of the original result after initialization so we can return that if the user
     // does not continue to the next step instead of returning any answers they might have given before
     // closing or cancelling
-    
     var originalResult: ORKStepResult!
     var userHasContinued = false
     
+    // We use a flag to track whether viewWillDisappear has been called because we run a check on
+    // viewDidAppear to see if we have any textFields in the tableView. This check is done after a delay,
+    // so we need to track if viewWillDisappear was called during the delay
+    var isViewDisappearing = false
```

Comment:
It reads a bit better to track `isVisible` rather than `isNotVisible`. Could you please change this? Thanks!